<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sensor</class>
 <widget class="QWidget" name="Sensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="l_Dest">
        <property name="text">
         <string>Répertoire Destination:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_Type">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QPushButton" name="btn_Annuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="le_Source"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_Dest"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="l_Ext">
        <property name="text">
         <string>Extensions</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="btn_ParcDest">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cb_Groupes"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Groupe</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLineEdit" name="le_ext">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>*.*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="btn_Valider">
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="btn_ParcSource">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_Name"/>
      </item>
      <item row="1" column="8">
       <widget class="QToolButton" name="Btn_Modifier">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icones/CadenasFerme.jpg</normaloff>
          <normalon>:/icones/CadenasOuvert.jpg</normalon>:/icones/CadenasFerme.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Répertoire source :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cb_Type">
        <item>
         <property name="text">
          <string>Fichiers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dossier</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTextEdit" name="te_Desc">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>71</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Description</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
